front/context/package.json

```diff
@@ -18,7 +18,7 @@
     "test": "echo skipped"
   },
   "dependencies": {
-    "@polkadot/ui-keyring": "^0.47.1",
+    "@polkadot/ui-keyring": "^0.48.1",
```

Comment:
since we're on beta for the rest of the polkadot-js suite, this should also be `0.49.0-beta.1`. fine to do in a later PR though.